docs/security/Ch-SecurityIssues.rst

```diff
@@ -0,0 +1,87 @@
+###############
+Security Issues
+###############
+
+
+
+This page describes how to report EdgeX Foundry security issues and how they are handled. 
+
+======================
+Security Announcements
+======================
+Join the edgexfoundry-announce group at: https://groups.google.com/d/forum/edgexfoundry-announce) 
+for emails about security and major API announcements.
+
+=======================
+Vulnerability Reporting
+=======================
+
+The EdgeX Foundry Open Source Community is grateful for all security reports made by users and security researchers. 
+All reports are thoroughly investigated by a set of community volunteers.
+
+.. _security_issue_template: https://github.com/edgexfoundry/edgex-go/blob/master/.github/ISSUE_TEMPLATE/4-security-issue-disclosure.md.
+To make a report, please email the private list: security-issues@edgexfoundry.org, providing as much detail as possible.
+Use the security issue template:  `security_issue_template`_.
```

Comment:
This template will need to be adjusted as it currently says to not report using github and instead use the vulnerability reporting procedure, which is what this document is supposed to be ... can you say recursion? :smiley: 
If the template will be adjusted in another PR that's fine, just calling this out